helm/secrets-provider/templates/secrets-provider.yaml

```diff
@@ -27,8 +27,17 @@ spec:
       containers:
       - image: {{ .Values.secretsProvider.image }}:{{ .Values.secretsProvider.tag }}
         imagePullPolicy: {{ .Values.secretsProvider.imagePullPolicy }}
+        {{- if .Values.environment.conjur.authnJWT.projectToken}}
+        volumeMounts:
+          - name: jwt-token
+            mountPath: /var/run/secrets/tokens
+        {{- end }}
         name: {{ .Values.secretsProvider.name }}
         env:
+        {{- if .Values.environment.conjur.authnJWT.projectToken}}
+        - name: JWT_TOKEN_PATH
```

Comment:
Using an environment variable here for the Secrets Provider "standalone" or "app" mode Helm chart is okay, but when the Secrets Provider is run in any other mode, this should be configured as a Pod Annotation. Is there a plan to add support to Secrets Provider so that this can be configured as a Pod Annotation?

Reply:
Actually, I thought we wouldn't need this setting, and the token would be hard-coded in the Secrets Provider code?

Reply:
It will be hardcoded  to defaults in documentation and helm charts but if client for any reason want to change he can

Reply:
We talked about not adding it as annotation not?